packages/server-wallet/src/wallet/store.ts

```diff
@@ -268,10 +268,13 @@ export class Store {
     const toMoverIndex = toState.turnNum % toState.participants.length;
     const toMover = toState.participants[toMoverIndex].signingAddress;
 
-    const turnNumCheck = _.isEqual(toState.turnNum, fromState.turnNum + 1);
-    if (!turnNumCheck) {
-      const VALIDATION_ERROR = `Turn number check failed.`;
-      logger.error({fromState, toState, error: Error(VALIDATION_ERROR)}, VALIDATION_ERROR);
+    if (fromState.appDefinition !== constants.AddressZero) {
```

Comment:
This predicate should not be inlined. There should be an `isLedgerChannel(signedState)` function that is called here.

Reply:
I actually specifically did not do this because I think this is a valid condition regardless of the type of channel — if there is no app definition then I don't think it matters to do turn number verification.

Reply:
Oh, fair. I guess it should be a `isNullApp` function but that seems ok.